pymc_marketing/mmm/events.py

```diff
@@ -278,10 +278,10 @@ class HalfGaussianBasis(Basis):
 
         import matplotlib.pyplot as plt
         from pymc_marketing.mmm.events import HalfGaussianBasis
-        from pymc_extras.priors import Prior
+        from pymc_extras.prior import Prior
```

Comment:
@juanitorduz, I found the issue. A bit of a facepalm moment. Excellent point to call it a night. Fingers crossed we see the plots in the docs now.